### PR TITLE
Adding omicverse

### PR DESCRIPTION
**Pull request by Starlitnightly**
Add the omicverse package.

(See https://github.com/Starlitnightly/omicverse/ for details.)

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Starlitnightly** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by Starlitnightly** [comment]
@conda-forge/help-python ready for review!

**Comment by Starlitnightly** [comment]
@ocefpaf Dear reviewer,

I kindly request you to review my pull request and merge my changes into the main repository. I am confident that these modifications will have a positive impact on the project's development.

If you have any further questions or suggestions, please feel free to ask. I am eager to further improve and refine the code.

Thank you again for your patience and support!

Best regards,
Zehua

**Comment by Starlitnightly** [comment]
@mwcraig Could you review my pull request🥹? Thank you again for your patience and support! 

Zehua

File: recipes/omicverse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1cab32bc2e87dc3d70785335480d2e70834cf66461daf504f33eadb265833cb0
+
+build:
+  skip: true  # [win or py<38]
```
Comment (mwcraig) [inline recipes/omicverse/meta.yaml:13]:
```suggestion
  python: noarch
```

File: recipes/omicverse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - flit >=3.2,<4
+    - python
```
Comment (mwcraig) [inline recipes/omicverse/meta.yaml:20]: A Python version restriction is required here for `noarch` packages
```suggestion
    - python >=3.8
```

File: recipes/omicverse/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    - pip
+    
+  run:
+    - python
```
Comment (mwcraig) [inline recipes/omicverse/meta.yaml:24]: Also need python version here for `noarch`
```suggestion
    - python >=3.8
```

**Review by mwcraig — CHANGES_REQUESTED** [review]
Thanks for opening this pull request, @Starlitnightly and congrats on your first package in conda-forge!

I think that you can make this a `noarch` project -- this saves quite a bit of resources on our CI compared to a project that must be compiled.

See below for the specific changes to make. When you have made the changes, please add a comment here and mention me.

File: recipes/omicverse/meta.yaml (hunk 1 above)
Comment (Starlitnightly) [reply recipes/omicverse/meta.yaml]: Hi, @mwcraig 

Thank you for your suggestion. I have updated this project to be a `noarch` project.

Zehua

**Comment by mwcraig** [comment]
Thanks for the updates (and sorry about the error in my suggestion to switch to noarch). I'm merging even though there is a windows failure because this is a noarch package.

**Review by mwcraig — APPROVED** [review]

**Pull request merged**